Tests/Tests/CrashLoggingTests.swift

```diff
@@ -70,7 +77,12 @@ class CrashLoggingTests: XCTestCase {
     func testWhenRunningOniOSThenEventsAreSentWithApplicationState() throws {
         let crashLogging = try CrashLogging(dataProvider: mockDataProvider).start()
         let event = crashLogging.beforeSend(event: Event(level: .debug))
+        #if SWIFT_PACKAGE
+        // When building for SPM, there is no active application for the tests.
+        XCTAssertEqual("unavailable", event?.tags?["app.state"])
```

Comment:
I don't know if this is just an SPM thing, or if it's a change in recent versions of Xcode.

Reply:
I don't know the answer either, but I would guess it's an SPM thing, something to do with Linux compatibility maybe? When running the tests for a package with `swift test`, for example, the Simulator doesn't load. Also, in the template projects, tests are simply files in a folder, there's no mention of an application in the `Package.swift` or in the project Xcode generates to work on the package. 🤔